<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettingsDialog</class>
 <widget class="QDialog" name="GameSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New game settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
   <item row="0" column="0">
    <widget class="QLabel" name="difficultyLevelLabel">
     <property name="text">
      <string>Difficulty level:</string>
     </property>
     <property name="buddy">
      <cstring>difficultyLevelEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="difficultyLevelEdit"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="settingsPanel">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="rowCountLabel">
        <property name="text">
         <string>Row count:</string>
        </property>
        <property name="buddy">
         <cstring>rowCountEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="rowCountEdit">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="columnCountLabel">
        <property name="text">
         <string>Column count:</string>
        </property>
        <property name="buddy">
         <cstring>columnCountEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="columnCountEdit">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mineCountLabel">
        <property name="text">
         <string>Mine count:</string>
        </property>
        <property name="buddy">
         <cstring>mineCountEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="mineCountEdit">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
